src/PuliPluginImpl.php

```diff
@@ -200,6 +251,13 @@ public function postInstall()
 
     private function initialize()
     {
+        if (!file_exists($this->autoloadFile)) {
+            $filesystem = new Filesystem();
+            // Avoid problems if using the runner before autoload.php has been
+            // generated
```

Comment:
Hm, can you outline when this exactly happens? I haven't seen any warning during my tests